frontend/tests/controllers/ContestListv2Test.php

```diff
@@ -0,0 +1,259 @@
+            $contestListPayload['contests']
+        );
+
+        $this->assertEqualsCanonicalizing(
+            $contestListPayloadAliases,
+            [
+                'current' => [
+                    'current-private',
+                    'current-public',
+                ],
+                'future' => [
+                    'future-private',
+                    'future-public',
+                ],
+                'past' => [
+                    'past-private',
+                    'past-public',
+                ]
+            ]
+        );
+    }
+
+    public function testPrivateContestsForNonInvitedUser() {
+        [
+            'contestData' => $contestData
+        ] = $this->createContests();
+
+        // Create user that wont be invited
+        ['identity' => $nonInvitedUserIdentity] = \OmegaUp\Test\Factories\User::createUser();
+
+        // Logging user
+        $login = self::login($nonInvitedUserIdentity);
+
+        $contestListPayload = \OmegaUp\Controllers\Contest::getContestListDetailsv2ForTypeScript(
+            new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+            ])
+        )['smartyProperties']['payload'];
+
+        $contestListPayloadAliases = $this->extractAliases(
+            $contestListPayload['contests']
+        );
+
+        $this->assertEqualsCanonicalizing(
+            $contestListPayloadAliases,
+            [
+                'current' => [
+                    'current-public',
+                ],
+                'future' => [
+                    'future-public',
+                ],
+                'past' => [
+                    'past-public'
+                ]
+            ]
+        );
+    }
+
+    public function testPrivateContestsForSystemAdmin() {
+        [
+            'contestData' => $contestData
+        ] = $this->createContests();
+
+        // Create admin user (system admin)
+        ['user' => $user, 'identity' => $adminUserIdentity] = \OmegaUp\Test\Factories\User::createAdminUser();
+
+        // Logging user
+        $login = self::login($adminUserIdentity);
+
+        $contestListPayload = \OmegaUp\Controllers\Contest::getContestListDetailsv2ForTypeScript(
+            new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+            ])
+        )['smartyProperties']['payload'];
+
+        $contestListPayloadAliases = $this->extractAliases(
+            $contestListPayload['contests']
+        );
+
+        $this->assertEqualsCanonicalizing(
+            $contestListPayloadAliases,
+            [
+                'current' => [
+                    'current-private',
+                    'current-public',
+                ],
+                'future' => [
+                    'future-private',
+                    'future-public',
+                ],
+                'past' => [
+                    'past-private',
+                    'past-public',
+                ]
+            ]
+        );
```

Comment:
```suggestion
        $this->assertEqualsCanonicalizing(
            [
                'current' => [
                    'current-private',
                    'current-public',
                ],
                'future' => [
                    'future-private',
                    'future-public',
                ],
                'past' => [
                    'past-private',
                    'past-public',
                ]
            ],
            $contestListPayloadAliases,
        );
```

en la documentación dicen que el primer parámetro es lo que se espera y el segundo lo que se obtuvo: https://phpunit.readthedocs.io/en/9.3/assertions.html#assertequalscanonicalizing

igual en los demás lugares